cloudify_cli/commands/ssh.py

```diff
@@ -28,6 +30,17 @@
 from cloudify_cli.exceptions import CloudifyCliError
 
 
+def _open_interactive_shell(host_string, command=''):
+    """Used as fabric's open_shell=True doesn't work well.
+    """
+    cmd = ['ssh', '-t']
+    cmd.append(host_string)
+    cmd.extend(['-i', os.path.expanduser(utils.get_management_key())])
+    if command:
```

Comment:
create the cmd list like this:
cmd = [
    'ssh', '-t', host_string,
    '-i', os.path.expanduser(utils.get_management_key()),
]
You want to do the most when you create the list,
You want to modify the list as little as possible...